packages/babel-plugin/__tests__/styled-macro/__fixtures__/label.js

```diff
@@ -3,30 +3,4 @@
 import styled from '@emotion/styled/macro'
 import { jsx } from '@emotion/react'
 
-class Thing {
```

Comment:
I've split this file into several, smaller ones. Each new test added to this has been changing generated source maps for each test case (so it has affected unaffected cases) and it was really noisy.